frontend_tests/puppeteer_tests/drafts.ts

```diff
@@ -40,7 +40,7 @@ async function create_stream_message_draft(page: Page): Promise<void> {
     await page.keyboard.press("KeyC");
     await page.waitForSelector("#stream-message", {visible: true});
     await common.fill_form(page, "form#send_message_form", {
-        stream_message_recipient_stream: "all",
+        stream_message_recipient_stream: "Denmark",
```

Comment:
Why are we using "specific stream" @YashRE42?

Reply:
Apparently, no reason. I dropped this commit.
The long answer is probably that there was a point in time when the implementation couldn't handle a stream that didn't exist ("all") and so I switched the tests but never dropped the commit when the handling was improved.